apps/angular/src/app/features/anime-dashboard/components/anime-table/anime-table.component.html

```diff
@@ -0,0 +1,53 @@
+@if (animeList) {
+	@if (animeList.length === 0){
+		<p class="anime-list__empty-message"> No animes to display. </p>
+	} @else {
+		<table mat-table [dataSource]="animeList" [trackBy]="trackByAnimeId">
+			<!-- Image Column -->
+			<ng-container matColumnDef="image">
+				<th mat-header-cell *matHeaderCellDef> Image </th>
+				<td mat-cell *matCellDef="let element">
+					@if (element.imageSrc){
+						<img [src]="element.imageSrc" class="anime-image" alt="{{element.titleEnglish}}" >
```

Comment:
```suggestion
						<img [src]="element.imageSrc" class="anime-image" [alt]="element.titleEnglish" >
```
It is better to use property bindings than property interpolation